var/spack/repos/builtin/packages/libsodium/package.py

```diff
@@ -29,14 +29,21 @@ class Libsodium(Package):
     """Sodium is a modern, easy-to-use software library for encryption,
     decryption, signatures, password hashing and more."""
     homepage = "https://download.libsodium.org/doc/"
-    url      = "https://download.libsodium.org/libsodium/releases/libsodium-1.0.3.tar.gz"
```

Comment:
I would keep this `url`. It is used when you run `spack versions` or `spack checksum`.